### DISCUSSION TITLE
Disable a field based on its row ordering index

### DISCUSSION THREAD

**Discussion by husam-tariq** [awcodes/filament-table-repeater#135] [Q&A] [answered]
I’m trying to disable a field based on its row ordering index, I’m wondering if there is any way to do that?

**Comment by awcodes** [comment, marked as answer]
Not sure of a good way to do this. Maybe check the discord for doing it in filament repeater. It's been discussed before. This plugin is just an abstraction on top of filament's repeater with a different view.
